library/src/main/java/com/panoramagl/hotspots/PLHotspot.java

```diff
@@ -247,24 +250,27 @@ public void getRect(PLRect rect) {
 
     @Override
     public void setX(float x) {
-        if(this.getX() != x) {
-	    super.setX(x);
-	    hasChangedCoordProperty = true;
-	}
+        if (this.getX() != x) {
+            super.setX(x);
+            hasManuallySetPosition = true;
+            hasChangedCoordProperty = true;
+        }
     }
 
     @Override
     public void setY(float y) {
-        if(this.getY() != y) {
-	    super.setY(y);
-	    hasChangedCoordProperty = true;
-	}
+        if (this.getY() != y) {
```

Comment:
I guess I should apply `ktLint` for pull requests

Reply:
I think there is some formatting strange behaviour happening, because i checked more than one time and did an auto-formatting from AS several times...